.github/workflows/python-app.yml

```diff
@@ -6,44 +6,42 @@ name: Robyn Python application
 
 on:
   push:
-    branches: [ "robynpy_release" ]
+    branches: ['robynpy_release']
   pull_request:
-    branches: [ "robynpy_release" ]
+    branches: ['robynpy_release']
 
 permissions:
   contents: read
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
-          matrix:
-            python-version: ["pypy3.9", "3.9", "3.10"]
-        
+      matrix:
+        python-version: ['pypy3.9', '3.9', '3.10']
+
     steps:
-    - uses: actions/checkout@v4
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v5
-      with:
-        python-version: ${{ matrix.python-version }}
-        cache: 'pip'
-    - name: Display Python version
-      run: python -c "import sys; print(sys.version)"
-        
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install flake8 pytest pytest-cov
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-    - name: Lint with flake8
-      run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude=./robyn_api/*.py
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude=./robyn_api/*.py
-    # - name: Test with pytest. Enable this once first set of tests are written
-    #   run: |
-# 
-#        pytest ./python/tests --doctest-modules --junitxml=junit/test-results.xml --cov=com --cov-report=xml --cov-report=html
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest pytest-cov
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Update requirements.txt to ./python/requirements.txt or python/requirements.txt and see if that fixes the dependencies issue.

Reply:
updated